src/shader/camera.rs

```diff
@@ -17,41 +37,26 @@ impl From<config::PresetCamera> for Camera {
         match cam {
             Perspective(persp) => persp.into(),
             //TODO(#4): This should be implemented as part of #4.
-            Custom {position, target} => unimplemented!(),
+            Custom(camera) => camera.into(),
         }
     }
 }
 
 impl From<config::Perspective> for Camera {
     fn from(persp: config::Perspective) -> Self {
-        //TODO(#4): This should be reimplemented properly as part of #4. These placeholder values
-        // are only meant to work for the desired angles of bigboi. The angles are slightly tilted.
-        // In the actual implementation they should be straight on.
+
+        // NOTE: PerspectiveLeft means point the camera to the left side of the x axis( negative x values ).
```

Comment:
Yes, this means that we put the camera on the left side. The camera is still pointed towards the center (0,0,0,1).

Reply:
The purpose of the comment is to distinguish the difference between pointing the camera on the left/right side of the model, and actually making the model look along its left or right side.

For example, given a front-facing model, if I want my model pointing to the right (to the +x values), I have to actually place the camera to the left of it.  In this case, the position of the camera, and the orientation of the model are different (one is left, the other is right).

Thus, for clarity purposes, I want to let the user know whether 'PerspectiveLeft' means "the model is pointed to the left" or "we are looking at the left side of the model" (since these mean different things) :) .

Reply:
I will change the comment to make my point more abbreviated